### DISCUSSION TITLE
test discussion

### DISCUSSION THREAD

**Discussion by bastelfreak** [choria-io/general#2] [Ideas]
test if I can create one. seems to work \o/

**Comment by ripienaar** [comment]
Yay, thanks @bastelfreak 👍

**Comment by bastelfreak** [comment]
I'm not sure if discussions can be closed in general, but I cannot do that

![bla](https://user-images.githubusercontent.com/341866/101620859-546bb080-3a15-11eb-8751-390d6b0a3117.png)

**Comment by ripienaar** [comment]
There's a "delete discussion" button on the right for me?
![Screenshot 2020-12-09 at 12 01 28](https://user-images.githubusercontent.com/82342/101621609-466a5f80-3a16-11eb-9a82-0653b64931a0.png)

**Reply by vjanelle** [reply]
I've got 'lock conversation', but also a collaborator on the project so likely have higher perms but not the same as yours.

**Comment by bastelfreak** [comment]
ah nope. looks like there are different permissions
![2020-12-09-120237_312x176_scrot](https://user-images.githubusercontent.com/341866/101621753-76b1fe00-3a16-11eb-9ac5-e23092153ad5.png)
